dfusion_rust_core/src/models/flux.rs

```diff
@@ -36,6 +41,31 @@ impl From<mongodb::ordered::OrderedDocument> for PendingFlux {
     }
 }
 
+impl From<Arc<Log>> for PendingFlux {
+    fn from(log: Arc<Log>) -> Self {
+        let mut bytes: Vec<u8> = log.data.0.clone();
+        PendingFlux {
+            account_id: pop_u16_from_log_data(&mut bytes),
+            token_id: pop_u8_from_log_data(&mut bytes),
+            amount: pop_u128_from_log_data(&mut bytes),
+            slot: pop_u256_from_log_data(&mut bytes),
+            slot_index: pop_u16_from_log_data(&mut bytes),
```

Comment:
I see `pop_u16_from_log_data` twice in here. Is it safe to assume that `pop` literally means pop (like in computer lingo) so that the information is forgotten after it is popped and the second call of this is just taking the next 16 bytes off the front?

Reply:
Yes, happy to rename to something that makes that more clear.